app/src/main/webapp/WEB-INF/jsp/tiles/views/home/publications.jsp

```diff
@@ -15,6 +15,17 @@
                     </div>
                 </div>
             </a>
+            <br/>
+            <a href="https://www.nature.com/articles/s41592-021-01102-w" target="_blank">
+                <div class="media-object margin-bottom-large">
+                    <div class="media-object-section middle">
+                        <h3><span class="icon icon-conceptual" data-icon="l"></span></h3>
+                    </div>
+                    <div class="media-object-section middle">
+                        <p>User-friendly, scalable tools and workflows for<br/> single-cell RNA-seq analysis, 29 March 2021.</p>
```

Comment:
```suggestion
                        <p>User-friendly, scalable tools and workflows for<br/> single-cell RNA-seq analysis <i>Nature Methods</i>, 29 March 2021.</p>
```

So that the journal is shown, as in the first case.